pkg/kubelet/volumes.go

```diff
@@ -132,8 +147,9 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.Pod) (kubecontainer.VolumeMap,
 			glog.Errorf("Could not create volume mounter for pod %s: %v", pod.UID, err)
 			return nil, err
 		}
+		//This should never be hit if there is a failure in mounter - probably can remove??
 		if mounter == nil {
-			return nil, errUnsupportedVolumeType
+			return nil, createVolumeTypeError(internal.Volume.Name)
```

Comment:
I guess this would be an unknown error because when the plugin is not found you usually get an error.